internal/connect/command_offset_alter_status.go

```diff
@@ -0,0 +1,73 @@
+package connect
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/kafka"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type alterStatusOut struct {
+	Id        string `human:"ID" serialized:"id"`
+	Phase     string `human:"Phase" serialized:"phase"`
+	Message   string `human:"Message,omitempty" serialized:"message,omitempty"`
+	AppliedAt string `human:"Applied at,omitempty" serialized:"applied_at,omitempty"`
+}
+
+func (c *offsetCommand) newGetAlterOffsetStatusCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:         "status",
```

Comment:
Our file naming convention for command implementations is to match the usage of the command. So in this case, we should rename this file to `command_offset_status.go`.